input/pagecontent/reference.md

```diff
@@ -3207,6 +3219,13 @@ Following [standard profiling rules established in FHIR](https://hl7.org/fhir/R5
   * performer[Practitioner] only Reference(PrimaryCareProvider)
   ```
 
+* Restrict the performer Reference datatype to the LiteralReference profile and its target types to Practitioner or PractitionerRole. Note that the first rule restricts the Reference datatype and the second rule restricts the types it can refer to:
```

Comment:
I think it would be better to split into two examples. Mixing these together creates the impression that the second statement somehow depends on the first.

Reply:
This example was specifically requested by Lloyd. In particular, he wasn't sure that the two only rules could work _together_ like this. So having them together is actually the point.

Reply:
Ok, leave it as-is then.